app/src/main/java/com/zeapo/pwdstore/pwgenxkpwd/CapsType.kt

```diff
@@ -5,5 +5,5 @@
 package com.zeapo.pwdstore.pwgenxkpwd
 
 enum class CapsType {
-    lowercase, UPPERCASE, TitleCase, Sentencecase, As_iS
```

Comment:
why this change?

Reply:
@msfjarvis:  because the labels are the same as values here, and the enum value is taken as eg `TitleCase.valueOf("Sentencecase")`. The string `Sentencecase` was too long and was getting clipped in portrait mode therefore the enum had to be changed together with the string value.

Reply:
Alright that makes sense